crates/walrus-service/src/test_utils.rs

```diff
@@ -2329,7 +2334,7 @@ pub mod test_cluster {
             Self {
                 epoch_duration: None,
                 test_nodes_config: None,
-                num_checkpoints_per_blob: Some(10),
+                num_checkpoints_per_blob: Some(100),
```

Comment:
With smaller value, I'm not able to trigger event catching up using event blob due to that reading event blob always returns non-existence due to delayed certified event processing.

Reply:
ok this makes sense to me, with event blobs we don't fire register event until it has been certified. Which means to be able to read the event blob, we need some buffer to allow nodes to have received those events to facilitate blob read.